flow/designs/asap7/megaboom/macro-placement.tcl

```diff
@@ -0,0 +1,74 @@
+set block [ord::get_db_block]
+set units [$block getDefUnits]
+
+set coreArea [$block getCoreArea]
+set xMin [$coreArea xMin]
+set yMin [$coreArea yMin]
+
+foreach uname [list \
+                    ] {
+
+  set inst [$block findInst $uname]
+
+  #$inst setOrient R0
+  #$inst setOrigin x y
+  #$inst setPlacementStatus FIRM
+}
+
+set x $xMin
+set y $yMin
+set y [expr $yMin - int(0.012 * $units)]
```

Comment:
@maliberty recommended using int and dbu.

mock-array uses round() or use dbu. w int you are rounding down and you will be off by 1nm suddenly. see mock-array for round(). this causes havoc in detailed routing.